src/needs-more-verification.js

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const core = require('@actions/core');
+const { isInRegistry, getFromRegistry, TEMPLATE_STATUS_APPROVED } = require('./registry');
+
+/**
+ * Criteria for needing additional verification:
+ * - published by third party developer (not under @adobe org)
+ *
+ *
+ * @param {string} npmPackage the name of the NPM package
+ * @returns {boolean}
+ */
+ function needsMoreVerification(npmPackage) {
+  return !npmPackage.startsWith('@adobe/');
+}
+
+(async () => {
+  try {
+    const myArgs = process.argv.slice(2);
+    const npmPackage = myArgs[0];
+
+    let existingTemplateApproved = false
+    if (isInRegistry(npmPackage)) {
+      const savedTemplate = getFromRegistry(npmPackage);
+      existingTemplateApproved = savedTemplate.status === TEMPLATE_STATUS_APPROVED
```

Comment:
I originally added this section so that if an update-template issue was opened on a third-party template that was already approved, it wouldn't need to be manually approved again. I see that you're saying that we may indeed want third-party templates to be manually approved every time they're updated. I'll take out this section.

Reply:
@AmyJZhao , a decision how to proceed (do we need an additional verification step or not) should be made by `.github/workflows/needs-more-verification.yml`, maybe we will introduce an allow-list for some template owners in the future, for example, to expedite their template updates